index.html

```diff
@@ -1327,8 +1327,11 @@
 information can be extremely sensitive (because it's identifying, because it
 allows for in-person intrusions, because it can reveal detailed information
 about a person's life) but it might also be public and not sensitive at all, or
-it might be low-enough granularity that it is much less sensitive for many
-people.
+it might be low-enough granularity that it is less sensitive for many
+people. Reducing granularity of location information may not hide precise location information from
+an [=actor=], however, particularly if location information is repeatedly requested over time or if
+the [=actor=] has other relevant information about the [=person=] [<a
+data-cite="RFC6772#section-13.5">RFC6772</a>].
```

Comment:
I'd lead with the fact that this is a caveat. Either:

```suggestion
people. Beware that reducing granularity of location information may not hide precise location information from
an [=actor=], particularly if location information is repeatedly requested over time or if
the [=actor=] has other relevant information about the [=person=] [<a
data-cite="RFC6772#section-13.5">RFC6772</a>].
```

Or something like:

```suggestion
people. Beware that [=actors=] can usually recover more precise location information than is 
exposed by any individual function call, particularly if location information is
repeatedly requested over time or if
the [=actor=] has other relevant information about the [=person=] [<a
data-cite="RFC6772#section-13.5">RFC6772</a>].
```